src/backend/utils/adt/numeric.c

```diff
@@ -5906,8 +5906,11 @@ numeric_poly_sum(PG_FUNCTION_ARGS)
 #endif
 }
 
+/*
+ *	Final function for BIGINT datatype for TSQL Aggregates (SUM,AVG)
+ */
```

Comment:
Add more explanation about function here.

Reply:
Fixed in the latest commit